client/components/PubCollectionsListing/PubCollectionsListing.tsx

```diff
@@ -196,19 +226,32 @@ const PubCollectionsListing = (props: Props) => {
 		);
 	};
 
+	const emptyListPlaceholderText = scopeData.activePermissions.canManageCommunity
+		? 'Create a new Tag with this name'
+		: 'No Collections match this search.';
+
+	const searchPlaceholderText = scopeData.activePermissions.canManageCommunity
+		? 'Search for Collections (or create a Tag)'
+		: 'Search for Collections';
+
+	const createNewItemProps = scopeData.activePermissions.canManageCommunity
```

Comment:
One thing that occurs to me is that if there's no `emptyListPlaceholderText` for Community managers, it might make more sense to fold all three of these variable declarations into a single choice between two props objects:

```ts
const createNewItemProps = canManageCommunity
    ? { createNewItemFromQuery, createNewItemRenderer, searchPlaceholderText }
    : { emptyListPlaceholderText, searchPlaceholderText };
```

(omitting values for clarity)

Reply:
there is an type error thrown requesting the emptyListPlaceholder prop if i remove it